src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/InstanceHelper.cs

```diff
@@ -155,5 +151,33 @@ public static (string LastChangedBy, DateTime? LastChanged) FindLastChanged(Inst
 
             return (lastChangedBy, lastChanged);
         }
+
+        /// <summary>
+        /// Replaces central texts in instance with values from text resource.
+        /// </summary>
+        /// <param name="instances">list of instances</param>
+        /// <param name="textResources">list of text resources</param>
+        /// <param name="language">the language</param>
+        /// <returns>list of instances with updated texts</returns>
+        public static List<MessageBoxInstance> ReplaceTextKeys(List<MessageBoxInstance> instances, List<TextResource> textResources, string language)
```

Comment:
this will result in app title to be shown as the repo name if ServiceName not available in language. Today "nb" is returned if the the title is not found.. Should this functionality be implemented here as well?

Reply:
@SandGrainOne  Yup. It would require us to retrieve the norwegian text file if we cant find the english or nynorsk one. But it might be a better alternative than showing the repo name